ionoscloud/resource_dbaas_mariadb_cluster.go

```diff
@@ -0,0 +1,245 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	dbaasService "github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/dbaas"
```

Comment:
`dbaasService` is not quite idiomatic for custom go import declarations
we should do some refactoring to import all we need only once from the `services` package